src/main/java/net/consensys/orion/cmd/Orion.java

```diff
@@ -130,7 +131,6 @@
   private HttpServer clientHTTPServer;
 
   public static void main(final String[] args) {
-    log.info("starting orion");
```

Comment:
Moved this to another place to avoid logging that we are starting Orion when we aren't actually starting it.